chapters/division_of_work/divisiton_of_work.tex

```diff
@@ -0,0 +1,56 @@
+\chapter{Division of work}\label{ch:division_of_work}
+This chapter describes the division of work on the application and the document.
+The division of work on the application is presented in \autoref{tab:division_of_work_on_the_application} and the division of work on the document is presented in \autoref{tab:division_of_work_on_the_document}.
+It is important to note that all parts of the project were worked on by both authors.
+Each line in both the application source code and the document written by one of the authors was reviewed by the other author in a pull request.
+Moreover both authors made bug fixes and small changes to all parts of the project.
+These tables only show who the main author of each part was.
+
+\begin{table}[h]
```

Comment:
we can add a screenshot from github https://github.com/Non-Euclidean-World/Hyper/graphs/contributors
It's quite remarkable that we have exactly the same # of commits
![image](https://github.com/Non-Euclidean-World/Thesis/assets/54417806/c862901c-66ac-42d0-8552-74a04bf9c7a2)